src/javaharness/java/arcs/api/BUILD

```diff
@@ -9,15 +9,15 @@ load("@com_google_j2cl//build_defs:rules.bzl", "j2cl_library")
 load("@build_bazel_rules_android//android:rules.bzl", "android_library")
 
 android_library(
-    name = "api",
+    name = "api-android",
```

Comment:
Why the '-android' suffixes here (and below)? Is this commonly done, I haven't seen before.

Reply:
I expect only general Java library without suffix. Android versions should end with -android.

Reply:
This is fine because it aligns with the -j2cl target being web.